src/generateHTML.js

```diff
@@ -0,0 +1,56 @@
+function generateHTML(team) {
+  // console.log(team);
+  let html = `<!DOCTYPE html>
+<html lang="en">
```

Comment:
Consider template stringing your logic here to ensure all of your indentations carry over as expected :-)